components/nimbus/old-nimbus-sdk-rust-code-README.md

```diff
@@ -0,0 +1,17 @@
+# Experiments SDK
```

Comment:
I reckon we could probably just delete this file.

Reply:
Done.